package.json

```diff
@@ -118,6 +118,32 @@
         "mac": "shift+f5",
         "when": "editorLangId == python"
       }
+    ],
+    "taskDefinitions": [
+			{
+				"type": "austin",
+				"required": [
+					"file"
+				],
+				"properties": {
+					"file": {
+						"type": "string",
+						"description": "The python file path to profile."
+					},
+					"args": {
+						"type": "array",
+						"description": "Optional list of arguments to the task"
+					},
+          "austinArgs": {
+						"type": "array",
+						"description": "Optional list of austin command-line options"
+					},
+          "profileName": {
+            "type": "string",
+            "description": "Name of the generated profile"
+          }
+				}
+			}
```

Comment:
Formatting looks a bit off and not in line with the rest of the source

Reply:
it had some tabs in there for some reason.